barcampgb.org

```diff
@@ -0,0 +1 @@
+Subproject commit 8ae7da9b8bfbfc64e32d7d83f9d693bc1d606f42
```

Comment:
Sorry, not even sure what I did. I cant find more info on it let alone delete it...